src/engine/SCons/Tool/MSCommon/vs.py

```diff
@@ -198,6 +198,17 @@ def reset(self):
 # Tool/MSCommon/vc.py, and the MSVC_VERSION documentation in Tool/msvc.xml.
 
 SupportedVSList = [
+    # Visual Studio 2019
+    VisualStudio('14.2',
+                 vc_version='14.2',
+                 sdk_version='10.0A',
+                 hkeys=[],
+                 common_tools_var='VS160COMNTOOLS',
+                 executable_path=r'Common7\IDE\devenv.com',
+                 batch_file_path=r'VC\Auxiliary\Build\vcvars32.bat',
+                 supported_arch=['x86', 'amd64', "arm"],
```

Comment:
Don't know if this gets used anywhere but should include arm64 (as should the 2017 version from this is copied)

Reply:
All these platforms are optional with VS2017 and 2019. And you can even add your own custom platforms... Not sure what list would be appropriate.

Reply:
including arm64 would be appropriate.